### PR TITLE
Make deletion of events more efficient.

### PR DESCRIPTION
**Pull request by floitsch**
Keep one day of events or 100 events (whichever is more).

### CODE REVIEW

File: public/supabase_broker/squashed.sql [hunk 1]
```diff
@@ -419,13 +435,21 @@ $$;
 
 ALTER FUNCTION "toit_artemis"."insert_pod"("_pod_id" "uuid", "_pod_description_id" bigint) OWNER TO "postgres";
 
-CREATE OR REPLACE FUNCTION "toit_artemis"."max_event_age"() RETURNS interval
+CREATE OR REPLACE FUNCTION "toit_artemis"."max_events"() RETURNS integer
+    LANGUAGE "sql" IMMUTABLE
+    AS $$
+    SELECT 100;
```
Comment (kasperl) [inline public/supabase_broker/squashed.sql:441]: Can we make this a proper number like 128 (power of two) or 101 (a prime)?

File: public/supabase_broker/squashed.sql [hunk 2]
```diff
@@ -419,13 +435,21 @@ $$;
...
+$$;
+
+ALTER FUNCTION "toit_artemis"."max_events"() OWNER TO "postgres";
+
+CREATE OR REPLACE FUNCTION "toit_artemis"."min_event_age"() RETURNS interval
     LANGUAGE "sql" IMMUTABLE
     AS $$
-    SELECT INTERVAL '30 days';
+    SELECT INTERVAL '1 day';
```
Comment (kasperl) [inline public/supabase_broker/squashed.sql:449]: Maybe it's just me, but I think this might need to be a bit higher. 3 days perhaps?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: public/supabase_broker/squashed.sql (hunk 1 above)
Comment (floitsch) [reply public/supabase_broker/squashed.sql]: done.

File: public/supabase_broker/squashed.sql (hunk 2 above)
Comment (floitsch) [reply public/supabase_broker/squashed.sql]: done.

**Pull request merged**